components/common/Makefile

```diff
@@ -0,0 +1,46 @@
+# ###############################################################################
+#
+#                       Knative-Kafka Common Makefile
+#
+# ###############################################################################
+
+# Project Directories
+BUILD_ROOT:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+BUILD_DIR:=$(BUILD_ROOT)/build
+
+# Application Names (Align With Knative Eventing Version)
+APP_NAME=knative-kafka-common
+APP_VERSION=latest
+
+# Build / Packaging / Deployment Configuration
+APP_PATH=konduit.docker.repositories.sap.ondemand.com/com.sap.c4konduit
```

Comment:
probably best to change this URL to a gcs repo of Kyma?

Reply:
Actually for the common component its no longer needed, was only needed to be compatible with SAP build systems.  So I removed it.